src/app/services/shared/globalConstantsRepository.ts

```diff
@@ -9,48 +20,130 @@ export class GlobalConstantsRepository
     private serviceBaseUrl;
     private pixelatedPlanetAPIUrl;
     private contactAPIUrl;
+    private analyticsAPIUrl;
     private subscribeAPIUrl;
     private feedbackAPIUrl;
     private locationsUrl;
+    private sessionGUID;
     private workshopTypesUrl;
     private photographersUrl;
+    private trackedLocation;
     private locations:ILocation[];
     private workshopTypes:string[];
     private photographers:IPhotographer[];
-    private locationMap: {[key: number]: ILocation} = {};
```

Comment:
why did we remove this?

Reply:
we don't use these. they were just lying around